src/util/domains.py

```diff
@@ -207,7 +207,11 @@ def _adapt_one_domain(cr, target_model, old, new, model, domain, adapter=None, f
     # pre-check domain
     if isinstance(domain, basestring):
         try:
-            eval_dom = expression.normalize_domain(safe_eval(domain, evaluation_context, nocopy=True))
+            if version_gte("saas~17.2"):
+                eval_dom = expression.normalize_domain(safe_eval(domain, locals_dict=evaluation_context))
```

Comment:
Why passing the `evaluation_context` as `locals_dict` in this case?

Reply:
iirc, if you pass it in the globals it will break.